### PR TITLE
Add grid splitter docs

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

File: controls/gridsplitter/overview.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description: Try now the Telerik GridSplitter for .NET MAUI enabling users to make a choice between two mutually exclusive options.
+position: 0
+slug: gridsplitter-overview
+---
+
+# .NET MAUI GridSplitter Overview
+
+Telerik UI for .NET MAUI GridSplitter is a control, which allows you to redistribute the space between columns or rows of a Grid control. By dragging the splitter, the target columns or rows will be resized.
+
+The GridSplitter control automatically detects the targeted columns/rows to resize, while dragging the control it starts to resize the columns/rows and redistributes space between columns/rows. You can control whether the GridSplitter resizes rows or columns, and which columns or rows are resized relative to the column or row for which the GridSplitter control is defined.
```
Comment (yordan-mitev) [inline controls/gridsplitter/overview.md:13]:
```suggestion
The GridSplitter control automatically detects the target columns or rows to resize. As you drag the control, it begins resizing and redistributing space between them. You can specify whether the GridSplitter resizes rows or columns, as well as which columns or rows are adjusted relative to the GridSplitter's defined position.
```

File: controls/gridsplitter/overview.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+![.NET MAUI GridSplitter Overview](images/gridsplitter-overview.png "GridSplitter Overview")
+
+## Key Features
+
+* [Resize behavior]({%slug %})&mdash;The GridSplitter allows you to control which columns or rows are resized relative to the column or row for which the GridSplitter control is defined.
+* [Resize direction]({%slug %})&mdash;You can control whether the GridSplitter control resizes rows or columns. 
+* [Visual states]({%slug gridsplitter-styling%}#visual-states)&mdash;You can use visual states to change the appearance of the control based on its state-such as when it's disabled and hovered.
```
Comment (yordan-mitev) [inline controls/gridsplitter/overview.md:21]:
```suggestion
* [Visual states]({%slug gridsplitter-styling%}#visual-states)&mdash;You can use visual states to change the appearance of the control based on its state&mdash;for example, when it's disabled or hovered.
```

File: controls/gridsplitter/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+1. [Set up your .NET MAUI application]({%slug maui-getting-started %}#step-1-set-up-your-net-maui-application).
+
+1. [Download Telerik UI for .NET MAUI]({% slug maui-getting-started %}#step-2-download-telerik-ui-for-net-maui).
+
+1. [Install Telerik UI for .NET MAUI]({%slug maui-getting-started %}#step-3-install-telerik-ui-for-net-maui).
+
+## Define the Control
+
+**1.** When the your .NET MAUI application is set up, you are ready to add a GridSplitter control to your page.
```
Comment (yordan-mitev) [inline controls/gridsplitter/getting-started.md:29]:
```suggestion
**1.** When your .NET MAUI application is set up, you are ready to add a GridSplitter control to your page.
```

File: controls/gridsplitter/configuration.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+slug: gridsplitter-configuration
+---
+
+# .NET MAUI GridSplitter Styling
+
+The GridSplitter provides a set of properties for customizing its default behavior.
+
+## Parent Element
+
+You can use the `ParentLevel` (`int`) property to specify the level of the parent grid to resize
```
Comment (yordan-mitev) [inline controls/gridsplitter/configuration.md:15]:
```suggestion
You can use the `ParentLevel` (`int`) property to specify the level of the parent grid to resize.
```

File: controls/gridsplitter/configuration.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+The following options are supported:
+
+* (Default) `Auto`&mdash;Determines whether to resize rows or columns based on its alignment (based on the value of the `HorizontalAlignment` and `VerticalAlignment` properties) and width compared to height
+* `Columns`&mdash;Resizes columns when dragging the splitter.
+* `Rows`&mdash;Resize rows when dragging the splitter.
+
+## Resize Behavior
+
+You can control whether the GridSplitter control resizes rows or columns by using the `ResizeBehavior` (`enum` of type `Telerik.Maui.Controls.RadGridSplitter.GridResizeBehavior`) property.
```
Comment (yordan-mitev) [inline controls/gridsplitter/configuration.md:29]: The description of Resize Behavior and Resize Directions is the same. To add more value, describe what makes them different, don't use the same sentence.

File: controls/gridsplitter/overview.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+Telerik UI for .NET MAUI GridSplitter is a control, which allows you to redistribute the space between columns or rows of a Grid control. By dragging the splitter, the target columns or rows will be resized.
+
+The GridSplitter control automatically detects the targeted columns/rows to resize, while dragging the control it starts to resize the columns/rows and redistributes space between columns/rows. You can control whether the GridSplitter resizes rows or columns, and which columns or rows are resized relative to the column or row for which the GridSplitter control is defined.
+
+![.NET MAUI GridSplitter Overview](images/gridsplitter-overview.png "GridSplitter Overview")
+
+## Key Features
+
+* [Resize behavior]({%slug %})&mdash;The GridSplitter allows you to control which columns or rows are resized relative to the column or row for which the GridSplitter control is defined.
```
Comment (yordan-mitev) [inline controls/gridsplitter/overview.md:19]:
```suggestion
* [Resize behavior]({%slug %})&mdash;The GridSplitter allows you to control which columns or rows are resized relative to the position of the splitter.
```

File: controls/gridsplitter/configuration.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+* `CurrentAndNext`&mdash;Resizes the current and next `Grid` columns or rows. For a horizontal GridSplitter, space is redistributed between the row that is specified for the GridSplitter and the next row that is below it. For a vertical GridSplitter, space is redistributed between the column that is specified for the GridSplitter and the next column that is to the right.
+* `PreviousAndCurrent`&mdash;Resize the previous and current `Grid` columns or rows. For a horizontal GridSplitter, space is redistributed between the row that is specified for the GridSplitter and the next row that is above it. For a vertical GridSplitter, space is redistributed between the column that is specified for the GridSplitter and the next column that is to the left.
+* `PreviousAndNext`&mdash;Resize the previous and next `Grid` columns or rows. For a horizontal GridSplitter, space is redistributed between the rows that are above and below the row that is specified for the GridSplitter. For a vertical GridSplitter, space is redistributed between the columns that are to the left and right of the column that is specified for the GridSplitter.
+
+Here is an example with `ResizeDirection` property set.
```
Comment (yordan-mitev) [inline controls/gridsplitter/configuration.md:38]:
```suggestion
Here is an example of how to set the `ResizeDirection` correctly.
```

Comment (yordan-mitev) [inline controls/gridsplitter/configuration.md:38]: I suggest moving the example to the previous section&mdash;`## Resize Direction`. It should be in the section that it supports.

File: controls/gridsplitter/styling.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+title: Styling
+page_title: .NET MAUI GridSplitter Documentation - Styling
+description: Set the border, background, and gripper color of the Telerik UI for .NET MAUI GridSplitter and customize its visual appearance.
+position: 4
+slug: gridsplitter-styling
+---
+
+# .NET MAUI GridSplitter Styling
+
+The GridSplitter allows you to customize its look using the following properties:
```
Comment (yordan-mitev) [inline controls/gridsplitter/styling.md:11]:
```suggestion
The GridSplitter control offers a range of styling options, allowing you to customize its appearance and adjust its visual states for different user interactions.

The GridSplitter allows you to customize its look using the following properties:
```

File: controls/gridsplitter/styling.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+* `BackgroundColor` (`Color`)&mdash;Specifies the background color of the control.
+* `Background` (`Brush`)&mdash;Specifies the background brush of the control.
+* `BorderColor` (`Color`)&mdash;Specifies the border color of the control.
+* `BorderBrush` (`Brush`)&mdash;Specifies the border brush of the control.
+* `BorderThickness` (`Thickness`)&mdash;Specifies the border thickness of the control.
+* `CornerRadius` (`Thickness`)&mdash;Specifies the border corner radius of the control.
+* `ContentPadding` (`Thickness`)&mdash;Specifies the padding of the content in the control.
+
+In addition you can style the gripper by using the `GripperColor` (`Color`) property.
```
Comment (yordan-mitev) [inline controls/gridsplitter/styling.md:21]:
```suggestion
In addition, you can style the gripper by using the `GripperColor` (`Color`) property.
```

File: controls/gridsplitter/styling.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+## Visual States
+
+You can use visual states to change the appearance of the control based on its state-such as when it's disabled and hovered.
```
Comment (yordan-mitev) [inline controls/gridsplitter/styling.md:25]:
```suggestion
You can use visual states to change the appearance of the control based on its state, for example, when it's disabled and hovered.
```

File: controls/gridsplitter/styling.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+The GridSplitter provides the following CommonStates visual states:
+
+* `Normal`&mdash;Applies when the GridSplitter is in normal state.
+* `MouseOver`&mdash;Applies when the mouse pointer is over the GridSplitter.
+* `Disabled`&mdash;Applies when the GridSplitter is disabled.
+
+This is the result of visual states applied:
```
Comment (yordan-mitev) [inline controls/gridsplitter/styling.md:33]:
```suggestion
This is the result of applying the visual states:
```

File: controls/gridsplitter/styling.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+![.NET MAUI GrdiSplitter Styling](images/gridsplitter-styling.gif)
+
+## Example
+
+Here is an example how to style the GridSplitter control and use the visual states.
```
Comment (yordan-mitev) [inline controls/gridsplitter/styling.md:39]:
```suggestion
Here is an example of how to style the GridSplitter control and use the visual states.
```

**Review by yordan-mitev — COMMENTED** [review]
You need to add the new folder to the `_config.yml`.

**Comment by didiyordanova** [comment]
@yordan-mitev the GridSplitter will be aded to the _config.yml when we merge data pager :)

**Comment by didiyordanova** [comment]
@yordan-mitev the GridSplitter is added into the _config.yml file

File: controls/gridsplitter/styling.md (hunk 8 above)
Comment (yordan-mitev) [reply controls/gridsplitter/styling.md:11]: Consider adding an introductory sentence before jumping to the properties.

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**